pyomo/gdp/disjunct.py

```diff
@@ -2,83 +2,118 @@
 #
 #  Pyomo: Python Optimization Modeling Objects
 #  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
-#  Under the terms of Contract DE-NA0003525 with National Technology and 
-#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain 
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
 #  rights in this software.
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
-from pyutilib.misc.indent_io import StreamIndenter
-
-from pyomo.core import *
-from pyomo.core.base.plugin import register_component
-from pyomo.core.base.constraint import (SimpleConstraint,
-                                        IndexedConstraint,
-                                        _GeneralConstraintData)
+import logging
+import sys
+from six import iteritems, itervalues
+from weakref import ref as weakref_ref
+
+from pyomo.util.modeling import unique_component_name
+from pyomo.util.timing import ConstructionTimer
+from pyomo.core import register_component, Binary, Block, Var, Constraint, Any
+from pyomo.core.base.component import (
+    ActiveComponent, ActiveComponentData, ComponentData
+)
+from pyomo.core.base.numvalue import native_types
+#=======
+#from pyomo.core import *
+#from pyomo.core.base.plugin import register_component
+#from pyomo.core.base.constraint import (SimpleConstraint,
+#                                        IndexedConstraint,
+#                                        _GeneralConstraintData)
+#>>>>>>> master
```

Comment:
I don't think that you mean to have this in here.

Reply:
Yup - that is completely developer notes (it is to remind me where we were importing things from while I remove `from x import *` from the codebase).